src/olympia/activity/tasks.py

```diff
@@ -21,15 +21,19 @@ def process_email(message, spam_rating, **kwargs):
             if header.get('Name', '').lower() == 'message-id':
                 msg_id = header.get('Value')
     if not msg_id:
-        log.error('No MessageId in message, aborting.')
-        log.error(message)
+        log.error('No MessageId in message, aborting.', extra={
+            'message_obj': message
+        })
         return
     _, created = ActivityLogEmails.objects.get_or_create(messageid=msg_id)
     if not created:
-        log.error('Already processed [%s], skipping' % msg_id)
-        log.error(message)
+        log.error('Already processed email [%s], skipping', msg_id, extra={
+            'message_obj': message
+        })
```

Comment:
I don't think this needs to be an error - a warning at best.

Reply:
https://github.com/mozilla/addons-server/pull/14380/commits/562ce30d070336b18d10bd9cb426c126633a423d